requirements.txt

```diff
@@ -9,4 +9,5 @@ torch
 accelerate
 sentence_transformers
 tqdm
-transformers
\ No newline at end of file
+transformers
+json
```

Comment:
I think json is already with python distribution, we don't need to install this. Integration test is failing for this.